_modules/loops.md

```diff
@@ -7,13 +7,87 @@ icon: fa-solid fa-arrow-rotate-left
 order: 5
 ---
 
-# Loops and Variables
+## What are Loops?
 
-## exercises
-1. Use a loop to turn on all the LEDs white.
+As a programmer, it's important to write code that isn't repetitive; it makes your program much easier to understand and speeds up the pace at which you can work. **Loops** are a fundamental programming tool we can use to eliminate repetitive code. Similar to functions, they can include as many commands inside them as you'd like. Unlike functions, they are able to _repeat those commands_ for as long as you want them to. We will explore two types of loops: **for loops** and **while loops**.
 
-1. Make the all the LEDs blink. On for 500ms then off for 1s, repeated forever.
+## For Loops
 
-1. Make a single LED circle around the board, you choose the speed. 
+**For loops** are loops that run the code inside them _a specified number of times_. Here's an example:
+
+```c
+for (int currentLed = 1; currentLed <= 4; currentLed++) {
+    leds_set_color(currentLed, 255, 0, 0);
+}
+```
+
+This code does exactly the same thing as we did in the variables module: it turns the first four LEDs on your board red. But it takes only three lines of code (in the last module it took seven)! Even better, we can turn on as many _more_ LEDs as we'd like without changing our loop hardly at all. 
+
+Let's break down how the for loop works.
+
+* We use the keyword `for` to tell the _compiler_ we are creating a for loop.
+* Inside the parenthesis, we have three commands:
+    1. First, we declare a variable called `currentLed` and initialize it to 1. This variable is called the **loop variable** because it controls the loop.
+    1. Second, we tell the board how long we want the _for loop_ to run for; that is, we want it to run until the `currentLed` is 4, then stop after turning on LED 4. This is called the **loop condition**.
```

Comment:
change "we want it to run until the `currentLed` is 4, then stop after turning on LED 4." 

to 

"we want it to run as long as `currentLed <= 4` is true.  Once currentLed is 5 or larger it won't run the loop again"

Reply:
Done